### DISCUSSION TITLE
Add this to winget

### DISCUSSION THREAD

**Discussion by ySergi** [mifi/lossless-cut#1981] [Ideas]
![image](https://github.com/mifi/lossless-cut/assets/138529583/4e4d85ba-692e-4df0-bbec-101392f54417)
A software like this should be uploaded to winget to make it quick to download and install, please add it to winget.

**Comment by mifi** [comment]
Unfortunately I don't want to maintain yet another distribution channel but if someone wants to, feel free. related #1279
